bundle/bundle.go

```diff
@@ -18,6 +18,8 @@ import (
 	"reflect"
 	"strings"
 
+	"github.com/open-policy-agent/opa/server/types"
```

Comment:
The bundle package should not depend on the server/types package. Something needs to be reorganized here.

Reply:
Defined a patch object struct in the bundle package to remove that dep. We could define it in some common lib if necessary.